src/tests/tables.test.js

```diff
@@ -20,3 +20,60 @@ describe("duplicate indices", () => {
     expect(errors).toEqual([]);
   });
 });
+
+describe("broken links", () => {
+  it("should not contain broken links", () => {
+    let errors = [];
+    let files = glob.sync("src/*.json");
+
+    let resources = {};
+
+    const walk_recursive = (resources, file, object, errors) => {
```

Comment:
How do you feel about extracting this helper method outside of the tests?

Reply:
Sure, that aligns well with a non-overlapping separation between `forEachFile()` and `forEachFileRecursive()`.